tokens-studio/figma-plugin
# Fix border sides application and removal

### Why does this PR exist?

Fixes #1829 



### What does this pull request do?



### Testing this change

- Apply a border to the top/left/right/bottom
- The change should appear
- Remove an individual border side value and the other border sides should be maintained
- Video: https://github.com/tokens-studio/figma-plugin/issues/1829



### Additional Notes (if any)
